scene/2d/animated_sprite.cpp

```diff
@@ -490,6 +490,34 @@ Point2 AnimatedSprite::get_offset() const {
 	return offset;
 }
 
+void AnimatedSprite::set_anchor(const Point2 &p_anchor) {
+
+	anchor = p_anchor;
+	anchor.x = CLAMP(anchor.x, -1, 1);
+	anchor.y = CLAMP(anchor.y, -1, 1);
```

Comment:
I don't think you should clamp the anchor, actually I think there might be a few uses where an unclamped anchor makes sense.